_locales/de/messages.json

```diff
@@ -1,14 +1,14 @@
 {
     "You should only install scripts from sources that you trust.": {
-        "message": "Sie sollten nur Skripte installieren, deren Quelle Sie trauen können.",
+        "message": "Installieren Sie dieses Benutzerskript nur, wenn Sie der Quelle vertrauen.",
```

Comment:
I highly suspect that your changes here will be overwritten, next time I pull data out of transifex.

Reply:
All these strings are fully in sync with the transifex data.

With one exception, I am still observing: `New user script ...` has a new key phrase without `SPACE` before `...`
And there are some additional phrases, that will be added to transifex, once the english original is uploaded properly. I'll have an eye on them! (sp?)